azurerm/internal/services/web/app_service_managed_certificate_resource.go

```diff
@@ -45,6 +46,17 @@ func resourceArmAppServiceManagedCertificate() *schema.Resource {
 				ValidateFunc: validate.CustomHostnameBindingID,
 			},
 
+			"ssl_state": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  string(web.SslStateIPBasedEnabled),
```

Comment:
You have to pick one, and it can only be applied here (the binding itself rejects a value here unless the thumbprint is also supplied at creation). Strictly speaking, this is `Required`, but we try to avoid introducing new properties as such. So it's a call one way or the other... This will only have a value for customers that have manually bound the cert to the hostname, and we can add an upgrade note to the changelog to cover this change?